Deprecate NodeJS 12 (#3286)
plone/volto: docs/source/upgrade-guide/index.md

## Upgrading to Volto 15.x.x

## Deprecating NodeJS 12

> Since April 30th, 2022 NodeJS 12 is out of Long Term Support by the NodeJS community, so

Reviewer: We should avoid ordinals in dates.

```suggestion
Since April 30, 2022, NodeJS 12 is out of Long Term Support by the NodeJS community.
```

> we are deprecating it in Volto 13.

Reviewer: ```suggestion
NodeJS 12 is deprecated in Volto 13.
```

> Please update your projects to a NodeJS LTS version (14 or 16 at the moment of this writing, 16 is recommended).

Reviewer: ```suggestion
Please update your projects to a NodeJS LTS version, where either 14 or 16 is supported at the moment of this writing.
Version 16 is recommended.
```
